schema.json

```
@@ -384,14 +410,19 @@
                 "ZA",
                 "ZM",
                 "ZW"
-            ]
+            ],
+            "type": "string"
         },
         "date": {
             "format": "date",
             "pattern": "[0-9]{4}-[0-1]{1}[0-9]{1}-[0-3]{1}[0-9]{1}",
             "type": "string"
         },
         "doi": {
+            "description": "The DOI of the work (not the resolver URL, i.e., 10.5281/zenodo.1003150, not http://doi.org/10.5281/zenodo.1003150).",
```

```suggestion
            "description": "The DOI of the work (i.e., 10.5281/zenodo.1003150, not the resolver URL http://doi.org/10.5281/zenodo.1003150).",
```

Fix linguistic scoping.